apps/storefront/src/pages/order/orders.ts

```diff
@@ -0,0 +1,149 @@
+import B3Request from '@/shared/service/request/b3Fetch';
+import { OrderStatusItem } from '@/types';
+import { convertArrayToGraphql } from '@/utils';
+
+export type CustomerOrderNode = {
+  node: {
+    orderId?: string;
+    createdAt: number;
+    totalIncTax?: number;
+    money?: string;
+    poNumber?: string;
+    status: string;
+    companyName?: string;
+    firstName?: string;
+    lastName?: string;
+  };
+};
+
+export interface GetCustomerOrders {
+  data: {
+    customerOrders: {
+      totalCount: number;
+      edges: Array<CustomerOrderNode>;
+    };
+  };
+}
+
+export type CompanyOrderNode = {
+  node: {
+    orderId?: string;
+    createdAt: number;
+    totalIncTax?: number;
+    money?: string;
+    poNumber?: string;
+    status: string;
+    firstName?: string;
+    lastName?: string;
+    companyInfo?: {
+      companyName: string;
+    };
+  };
+};
+
+export interface GetCompanyOrders {
+  data: {
+    allOrders: {
+      totalCount: number;
+      edges: Array<CustomerOrderNode>;
+    };
+  };
+}
+
+const allOrders = (data: CustomFieldItems, fn: 'allOrders' | 'customerOrders') => `
```

Comment:
is the plan to turn these into regular queries (i.e. not pushing variables through string literals) in a different PR?

or maybe they're not currently getting in the way?